package.json

```diff
@@ -30,7 +30,7 @@
     "extend": "~3.0.1",
     "forever-agent": "~0.6.1",
     "form-data": "~2.3.1",
-    "har-validator": "~5.0.3",
+    "har-validator": "5.1.0",
```

Comment:
Why are these packages version locked while others are not?

Reply:
Better to just follow upstream dependencies with no changes IMO.